### PR TITLE
Sample Backend Improvements

### PR DESCRIPTION
**Pull request by LucasVidal**
Removed rest-client dependency since it breaks with ruby 2.4. Using net/http.
Improved documentation and forwarded errors from Authy API directly.

https://github.com/authy/authy-sdk-backend/pull/3

### CODE REVIEW

**Review by suarezjulian — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
-env.rb
```
Comment (senekis) [inline .gitignore]: Please don't remove the `env.rb` file from the ignore

File: env.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# The api_key can be obtained by logging into the Authy dashboard and selecting the app for 
+# which you are authenticating users. 
+#
+# https://dashboard.authy.com/
+
+ENV["AUTHY_API_KEY"] = "YOUR_API_KEY"
```
Comment (senekis) [inline env.rb]: Use the `env.rb.sample` file for this, the idea is that the developers copy the sample file to env (ignored file) and update with their api_keys, without any risk to commit by mistake their credentials to GitHub

**Comment by LucasVidal** [comment]
Done!
@senekis thanks for that. 
Anything else?

**Review by senekis — APPROVED** [review]

**Comment by LucasVidal** [comment]
@senekis @TechFreak gracias!
Como sigue? Hago merge y borro branches? (no se si tengo permisos tampoco)

**Pull request merged**